React/Modules/RCTAlertManager.m

```diff
@@ -30,6 +30,7 @@ @interface RCTAlertManager()
 @implementation RCTAlertManager
 {
   NSHashTable *_alertControllers;
+  UIWindow *_window;
```

Comment:
We should not store `UIWindow` as an ivar (because it's not shared among instances of view controllers), right?
Instead, we should just remove a window instance from Screen instance on alert being dismissed. We can geet a particular window instance from presented view controller (or any mounted view) instance.

Reply:
@shergin You are right, it doesn't need to be an ivar, it can be replaced with local variable here. But in that case it also must be strongly captured by the action handler block of UIAlertController, otherwise the window gets destroyed while we are showing the alert controller. It looks like `UIAlertController` (or `UIViewController` in general) doesn't keep a strong reference to its window.